test/cases/ingress-nginx/test_rules.py

```diff
@@ -0,0 +1,37 @@
+from .. import case
+
+import json
+import os
+from pathlib import Path
+from unittest import skipUnless
+
+
+@skipUnless(
+    os.getenv("NGINX_FLAVOR", "") == "ingress-nginx",
+    "Test embedded sampling rules only for ingress-nginx",
+)
+class TestIngressNginxRules(case.TestCase):
+
+    def test_healthcheck(self):
+        # Healthcheck request MUST not be generated traces.
```

Comment:
```suggestion
        # Healthcheck request MUST not generated traces.
```
[Be you, be proud of you because you can be do what we want to do](https://www.youtube.com/watch?v=dh7O5BV47lU)